configs/top_down/alexnet/README.md

```diff
@@ -0,0 +1,20 @@
+# Imagenet classification with deep convolutional neural networks
```

Comment:
does top-down methods only differs by backbone?

Reply:
Yes. For most top-down methods, the network architectures are the main differences.

Reply:
🤔

Reply:
let's merge it as is, and change only when necessary